client/state/billing-transactions/types.ts

```diff
@@ -0,0 +1,103 @@
+import type { RawReceiptPurchase } from 'calypso/state/receipts/types';
+
+export interface IndividualReceipt {
+	receipt_id: number;
+	purchases: RawReceiptPurchase[];
+	display_price: string;
+	price_integer: number;
+	price_float: number;
+	currency: string;
+	is_gift_purchase: boolean;
+}
+
+export interface BillingTransaction {
+	id: string;
+	service: string;
+	amount: string;
+	tax_amount: string;
+	icon: string;
+	date: string;
+	desc: string;
+	org: string;
```

Comment:
Is each of these required fields guaranteed to be there?

Reply:
I looked at the endpoint to come up with this type and I'm pretty sure all of these are always set (some might be empty strings, but I think they're always strings).